src/Integration.Vsix/Suppression/LiveIssueFactory.cs

```diff
@@ -108,12 +108,17 @@ public LiveIssue Create(SyntaxTree syntaxTree, Diagnostic diagnostic)
                 return new LiveIssue(diagnostic, projectGuid, lineSpan.Path);
             }
 
-            var startLine = lineSpan.StartLinePosition.Line;
-            var additionalLineCount = lineSpan.EndLinePosition.Line - startLine;
-            var lineText = syntaxTree.GetText().Lines[startLine + additionalLineCount].ToString();
-            var sonarQubeLineNumber = lineSpan.StartLinePosition.Line + 1; // Roslyn lines are 0-based, SonarQube lines are 1-based
-
-            return new LiveIssue(diagnostic, projectGuid, lineSpan.Path, sonarQubeLineNumber, lineText); // Line-level issue
+            try
+            {
+                var lineText = syntaxTree.GetText().Lines[lineSpan.EndLinePosition.Line].ToString();
```

Comment:
The exception was being thrown from this line.

Miscellaneous - I've simplified the code slightly: `startLine` and `additionalLineCount` were just adding complexity, and disguising the fact that we are just picking the last line if the diagnostic spans more than one line.